tests/unit/amo/components/TestCategoryIcon.js

```diff
@@ -1,37 +1,36 @@
 import * as React from 'react';
-import { shallow } from 'enzyme';
 
 import CategoryIcon from 'amo/components/CategoryIcon';
-import Icon from 'amo/components/Icon';
+import { render, screen } from 'tests/unit/helpers';
 
 describe(__filename, () => {
   it('renders an Icon and sets CSS classes', () => {
-    const root = shallow(<CategoryIcon name="foo" color="1" />);
-    const icon = root.find(Icon);
+    render(<CategoryIcon name="foo" color="1" />);
 
-    expect(icon).toHaveLength(1);
-    expect(icon).toHaveClassName('CategoryIcon');
-    expect(icon).toHaveClassName('CategoryIcon-1');
+    const icon = screen.getByClassName('Icon');
+    expect(icon).toHaveClass('CategoryIcon');
+    expect(icon).toHaveClass('CategoryIcon-1');
   });
 
   it('allows a custom className', () => {
-    const root = shallow(<CategoryIcon name="bar" color="1" className="sup" />);
-    const icon = root.find(Icon);
+    render(<CategoryIcon name="bar" color="1" className="sup" />);
 
-    expect(icon).toHaveClassName('CategoryIcon');
-    expect(icon).toHaveClassName('CategoryIcon-1');
-    expect(icon).toHaveClassName('sup');
+    const icon = screen.getByClassName('Icon');
+    expect(icon).toHaveClass('CategoryIcon');
+    expect(icon).toHaveClass('CategoryIcon-1');
+    expect(icon).toHaveClass('sup');
   });
 
   it('passes alt-text to the Icon', () => {
-    const root = shallow(<CategoryIcon name="bar" color="1" alt="Alt text!" />);
+    render(<CategoryIcon name="bar" color="1" alt="Alt text!" />);
 
-    expect(root.find(Icon)).toHaveProp('alt', 'Alt text!');
+    expect(screen.getByText('Alt text!')).toBeInTheDocument();
```

Comment:
`screen.getByAltText`?

Reply:
While the `Icon` component uses the `alt` prop to configure this, it doesn't actually render an element with an `alt` attribute, so we cannot use `getByAltText` here.